0410199804101998.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "0410199804101998",
```

Comment:
This is not a valid gateway ID, please follow the installer instructions and use the one printed in the console.